src/main/java/core/basesyntax/GetArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface GetArea {
```

Comment:
Interface names should not be verbs. Consider renaming 'GetArea' to something that represents an entity or a capability like 'AreaCalculatable' or 'AreaProvider'.